types/node/index.d.ts

```diff
@@ -68,6 +68,18 @@ interface SymbolConstructor {
 }
 declare var Symbol: SymbolConstructor;
 
+// Node.js ES2017 and ESNEXT support
+interface String {
+    /** Pads the current string with another string (repeated, if needed) so that the resulting string reaches the given length. The padding is applied from the start (left) of the current string. */
+    padStart(targetLength?: number, padString?: string): string;
```

Comment:
Should we require the `targetLength` argument since the [official docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) and the [proposal](https://github.com/tc39/proposal-string-pad-start-end/blob/master/spec.md#stringprototypepadstart-maxlength---fillstring--) require it? 

Just tested it without any args or negative numbers and they seem to return the same result as using it with `targetLength = string.length`.

Reply:
I think this is a typings for node.js, not for general es2017, so if node.js accepts this method without any arguments, we should accept optional arguments.

Reply:
Shouldn't this already been provided by adding es2017.string to your libs in tsconfig? lib/es2017.string.d.ts already provides an interface to those methods.

Reply:
@eps1lon The methods in es2017.string.d.ts require at least first argument. In Node.JS the first argument is optional.

But I would agree that we should be compatible with standard. Then I'm removing additional typings for padStart and padEnd and only trimLeft and trimRight will be defined for Node.